src/components/BlockCard/BlockCard.tsx

```diff
@@ -24,7 +23,8 @@ export default function BlockCard(props: IProps) {
         </CardHeader>
         <CardContent>
           <Typography variant="caption" style={{ fontSize: "11px" }}>{block.hash}</Typography>
-          <Typography gutterBottom>{d}</Typography>
+          <Typography gutterBottom>{hexToDate(block.timestamp)}</Typography>
+          <Typography gutterBottom>{new Buffer(block.extraData.substring(2), "hex").toString()}</Typography>
```

Comment:
this could be another helper function `hexToString`

Reply:
agreed if you're doing that conversion quite a bit. doesn't have to be in this PR up 2 you!